Queueomatic/Client/Shared/NavMenu.razor

```diff
@@ -9,7 +9,7 @@
 
 <div class="container pt-3">
     <LogoComponent/>
-    <AuthorizeView>
+    <AuthorizeView Roles="User,Administrator">
```

Comment:
Not sure why should we have to check the roles but this prevents a logged-in user from seeing the `Authorized` section.

Reply:
This is in place because a participant is considered to be an "authenticated" user. However, the policy in the client side was not working correctly so this is now updated to work correctly.